core/app/models/spree/shipment.rb

```diff
@@ -163,7 +163,7 @@ def line_items
     def manifest
       # Grouping by the ID means that we don't have to call out to the association accessor
       # This makes the grouping by faster because it results in less SQL cache hits.
-      inventory_units.group_by(&:variant_id).map do |variant_id, units|
+      inventory_units.includes(:line_item).group_by(&:variant_id).map do |variant_id, units|
```

Comment:
Unused block argument - `variant_id`. If it's necessary, use `_` or `_variant_id` as an argument name to indicate that it won't be used.